### PR TITLE
Added steps for running debug script, cleaned up topic.

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by marialuzshin — APPROVED** [review]
LGTM!

**Review by bahui80 — APPROVED** [review]
LGTM

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 1]
```diff
@@ -4,42 +4,43 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: agent, mule, servers, monitor, notifications, external systems, third party, get status, metrics
 
-A `$MULE_HOME/bin/amc_debug` script is available to help you troubleshoot Runtime Manager agent installation issues.
+You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plugin or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
```
Comment (kbostrom) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]:
```suggestion
You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot Anypoint Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plug-in or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
```

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 2]
```diff
@@ -4,42 +4,43 @@ include::_attributes.adoc[]
...
 
-If you experience issues with the Runtime Manager agent plugin or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
+The `amc_debug` script performs the following:
```
Comment (kbostrom) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]:
```suggestion
The `amc_debug` script performs the following actions:
```

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 3]
```diff
@@ -4,42 +4,43 @@ include::_attributes.adoc[]
...
 
-== Connectivity Problems
+. Open a terminal window and change to the `$MULE_HOME/bin` folder.
+. Run the script:
++
+`./amc_debug`
++
+Include the `-heap` option to extract the heap in addition to the other data.
++
+After the script completes, the folder `$MULE_HOME/bin` contains a .zip file with a name similar to `snapshot_201910081712.zip`.
```
Comment (kbostrom) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]:
```suggestion
After the script finishes, the folder `$MULE_HOME/bin` contains a .zip file with a name similar to `snapshot_201910081712.zip`.
```
"completes" requires an object; "is complete" is OK. See style ref.

File: modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc [hunk 4]
```diff
@@ -4,42 +4,43 @@ include::_attributes.adoc[]
...
+. Send the .zip file to MuleSoft support engineers.
```
Comment (kbostrom) [inline modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc]: Earlier referred to as just "Support." Standardize?

**Review by kbostrom — APPROVED** [review]
Done!

**Pull request merged**